### PR TITLE
Add not found

### PR DESCRIPTION
**Pull request by Winkel2019**
Hi @AdrianValdes, I have added the NotFound page and changed a bit on login page. Have a look, let me know if anything needs to be fixed.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -44,6 +45,7 @@ export const App = () => {
         <Route path='/login' component={Login} />
         <Route path='/signup' component={SignUp} />
         <Route exact path='/' component={Home} />
+        <Route path='*' component={NotFound404} />
```
Comment (AdrianValdes) [inline src/App.jsx:48]: good that you used the property `path='*'` here and didn't leave it blank.

File: src/app/NotFound404.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  align-self: center;
+  background-color: rgb(61, 20, 15);
+  margin-top: 650px;
+  width: 300px;
+`;
+
+export const NotFound404 = (props) => (
+  <Main>
+    <GoHomeButton type='button' onClick={() => props.history.push('/')}>
+      Take Me Home!
```
Comment (AdrianValdes) [inline src/app/NotFound404.jsx:28]: :+1:  It is good that you used `props.history.push('/')`:

Maybe we could change the message: "Take me Home!" sounds... I don't know, not good enough :). Maybe you could see what other websites do when they show you a 404 page.

Comment (AdrianValdes) [reply src/app/NotFound404.jsx:28]: We could also use some kind of image on the 404 page. Could you check what I did on the `NoResults.jsx` component?
You can see it by searching something that doesn't exist like "asadsfd" :).
I used a SVG from [here](https://undraw.co/), what do you think about using another one for the 404 page? They have a lot of options.

**Review by AdrianValdes — CHANGES_REQUESTED** [review]
Hi @Winkel2019!
I added some comments, could you check them out and tell me what you think? Remember to pull to see what I did. 

If you are going to apply changes on this branch don't `--rebase` from main. You can pull from main and see what I did, but don't `rebase` the main to your `branch`. If you have any doubts about `git` ask me please.

File: src/app/NotFound404.jsx (hunk 2 above)
Comment (Winkel2019) [reply src/app/NotFound404.jsx:28]: I have a background image on the 404 page.

Comment (AdrianValdes) [reply src/app/NotFound404.jsx:28]: Remember to always re-request reviews.

**Review by AdrianValdes — APPROVED** [review]

**Pull request merged**